website/docs/gitops-run/get-started.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Get Started
+sidebar_position: 2
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Get Started with GitOps Run
+
+Gitops Run supports two different modes of operation - directly on a
+cluster, or as sandboxed sessions. The sandboxed sessions are intended
+for shared environments where multiple users are running their own
+sessions, whereas the direct mode is intended for a local cluster.
+
+## Direct mode
+
+### Prerequisites
+#### Required
+- Install the gitops CLI. See [the installation](../installation.mdx#gitops-cli)
+
+#### Optional 
+- This guide uses [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for demonstrations, but it is not required to use gitops run
+- The [flux CLI](https://fluxcd.io/flux/installation/) is the quickest way to generate resource definitions, but the files can also be created manually
+
+### Kubernetes cluster
+To get started with gitops run, you need a kubernetes cluster. There
+are many tools to set up a local cluster for test and development
+purposes.
+
+**Note:** this tutorial assumes you have full control of your cluster - we
+recommend a local cluster, but you can also use a remote cluster where
+you have full `cluster-admin` privileges.
+
+<Tabs>
+<TabItem value="kind">
+
+Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and run
+
+```bash
+kind create cluster
+```
+</TabItem>
+<TabItem value="k3d">
+
+Install [k3d](https://k3d.io/) and run
+
+```bash
+k3d cluster create mycluster
+```
+</TabItem>
+<TabItem value="minikube">
+
+Install [minikube](https://minikube.sigs.k8s.io/docs/start/) and run
+
+```bash
+minikube start
+```
+</TabItem>
+<TabItem value="docker desktop">
+
+Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and enable Kubernetes. Then run
+
+```
+kubectl config set-context docker-desktop
+```
+</TabItem>
+<TabItem value="other">
+
+Gitops Run works on any kubernetes platform, but to avoid accidents
+you have to explicitly white-list the context name.
+
+First, find the name of the context where you want to run `gitops run` - in this example, there's a cluster with the name "dev":
+
+```bash
+$ kubectl config get-contexts
+CURRENT   NAME                      CLUSTER                   AUTHINFO                  NAMESPACE
+*         dev                       dev                       dev
+```
+
+Then, for any `gitops run` command in this guide, you'll have to add the flag `--allow-k8s-context=dev`
+</TabItem>
+</Tabs>
+
+Before you continue, make sure `kubectl get nodes` returns a node which is `Ready`.
+
+### Git repository
+
+You need to set up a git repository to put your Gitops manifests
+in. Any git repository would do, for example create a new
+[github](https://github.com/new) repository and clone that.
+
+### Set up gitops run
+
+To start gitops run, point it at the directory you want to keep your
```

Comment:
To start GitOps Run

or may be

To start `gitops run`?

Reply:
Gonna do GitOps Run, as the command is still `gitops beta run`